### PR TITLE
Game buttons actions positions

### PR DESCRIPTION
**Pull request by deplanty**
- Add the `execute` method to the button: emit the signal is the button is not disabled;
- Set the button actions in separated methods: easier to call them;
- Replace and resize the buttons in the game view: the text stay in the boundaries of the button.

### CODE REVIEW

File: src/controller/game_controller.py [hunk 1]
```diff
@@ -162,49 +166,23 @@ def play_one_round(self):
...
                         elif event.key in [K_1, K_KP1]:
-                            print(1)
-                            card = self.deck.getCard()
-                            human.add_card(card, hand_idx)
-                            print("You : " + str(human.hands[hand_idx].hand))
-                            if human.hands[hand_idx].hand.is_burnt or human.hands[hand_idx].hand.is_black_jack:
-                                print("Is burnt or black jack")
-                                state = False
-
+                            self.view_game.buttons["card"].execute()
```
Comment (ldehaine) [inline src/controller/game_controller.py:169]: Just an idea here, for later maybe, I think we should make the button looked pressed when we press the corresponding keyboard key, and call execute() only when the key is released, exactly how the do it when we press the button with the mouse pointer

File: src/controller/game_controller.py [hunk 2]
```diff
@@ -242,39 +220,67 @@ def btn_card(self):
         Manage actions on card button
         """
 
+        card = self.deck.getCard()
+        self.human.add_card(card, self.hand_idx)
+        print("You : " + str(self.human.hands[self.hand_idx].hand))
+        if self.human.hands[self.hand_idx].hand.is_burnt or self.human.hands[self.hand_idx].hand.is_black_jack:
+            print("Is burnt or black jack")
+            self.state = False
```
Comment (ldehaine) [inline src/controller/game_controller.py:228]: I know that's not something you added with this PR, but I don't like this boolean state attribute, what is it supposed to mean?
"How are you?"
"I'm False!"
"Bruh, feel bad for you..."

So if you have an idea about this...

**Review by ldehaine — APPROVED** [review]
Some comments but non-blocking

File: src/controller/game_controller.py (hunk 2 above)
Comment (Hboni) [reply src/controller/game_controller.py:228]: I added that part and I agree that the naming is weird :upside_down_face: 
 I think we can rename `self.state` into `self.playing` or `self.playing_turn` ?

File: src/button/button.py [hunk 3]
```diff
@@ -15,6 +15,7 @@ class State(enum.Enum):
     normal = enum.auto()
     hover = enum.auto
```
Comment (Hboni) [inline src/button/button.py]: Not in your PR, but lack of parenthesis !?

**Review by Hboni — COMMENTED** [review]
I found other problems, but are related to another branch.

File: src/button/button.py (hunk 3 above)
Comment (deplanty) [reply src/button/button.py]: True

File: src/controller/game_controller.py (hunk 1 above)
Comment (deplanty) [reply src/controller/game_controller.py:169]: I made an issue of your idea.

**Review by Hboni — APPROVED** [review]

**Pull request merged**